tests/iso-volumes.spec.ts

```diff
@@ -37,3 +39,36 @@ test("upload and delete custom iso", async ({ page }) => {
   await page.getByText("Delete", { exact: true }).click();
   await page.getByText(`Storage volume ${isoName} deleted.`).click();
 });
+
+test("use custom iso for instance launch", async ({ page }) => {
+  test.skip(Boolean(process.env.CI), "github runners lack vm support");
```

Comment:
Do we need to skip this? We're just creating the instance, not starting it. Or would it not be able to even create it?

Reply:
Yes exactly, it fails to create the instance with type VM, see also this run: https://github.com/canonical/lxd-ui/actions/runs/6108398358